README.md

```diff
@@ -1,71 +1,135 @@
-# Serverless commercetools Plugin
+# commercetools Serverless Plugin
+
+[commercetools's](https://commercetools.com/) plugin for the [Serverless Framework](https://serverless.com) allows you to seamlessly integrate your serverless functions with commercetools' extensibility options.
+
+## Contents
+
+- [Features](#features)
+- [Install](#install)
+- [Configuration](#configuration)
+  - [API Extension](#api-extension)
+  - [Subscription](#subscription)
+- [Usage](#usage)
+- [Help](#help)
+- [Development](#development)
+- [License](#license)
 
-Serverless framework plugin that registers the deployed function as a commercetools API Extension or attaches it to a Subscription.
+## Features
 
-## Configuration and Deploy for AWS and GCP
+- Attach a newly deployed serverless function as a commercetools' API extension.
+- Create a new serverless function to process events from a commercetools' subscription.
+- Supports AWS and GCP
 
-1.  At the bottom of your function's serverless.yaml place:
-    `plugins:`
-    `- serverless-commercetools-plugin`
+## Install
 
-    Plugin install method 1 - Install it as a regular [npm module ](https://www.npmjs.com/package/serverless-commercetools-plugin) and reference it in plugins section of your serverless.yml shown above. Or use the serverless plugin install command [serverless plugin install --name pluginName](https://www.serverless.com/framework/docs/providers/aws/cli-reference/plugin-install/).
+```sh
+npm i --save-dev serverless-commercetools-plugin
+# or
+yarn add --dev serverless-commercetools-plugin
+```
 
-    Plugin install method 2 - In your serverless function create a folder called .serverless_plugins. Place the serverless-commercetools-plugin folder into .serverless_plugins with dependencies installed.
+Add the following plugin to your `serverless.yml`:
 
-2.  In your serverless.yaml under the service, directly below region, add environment vars for your commercetools client.
+```yaml
+plugins:
+  - serverless-commercetools-plugin
+```
 
-    Example:
+With the [serverless CLI](https://www.serverless.com/framework/docs/providers/aws/cli-reference/plugin-install/)
 
-             region: us-east-2
```

Comment:
So I'm a bit confused here, does all of the plugin data live in the provider section?